projects/core/src/user/store/actions/billing-countries.action.ts

```diff
@@ -21,6 +24,17 @@ export class LoadBillingCountries implements Action {
 
 export class LoadBillingCountriesFail implements ErrorAction {
   readonly type = LOAD_BILLING_COUNTRIES_FAIL;
+
+  constructor(error: ActionErrorProperty);
+  /**
+   * @deprecated Please use the `error` parameter with a non-null, non-undefined value.
+   *             Support for `null` or `undefined` will be removed in future versions,
+   *             along with the feature toggle `ssrStrictErrorHandlingForHttpAndNgrx`.
+   */
+  constructor(
+    // eslint-disable-next-line @typescript-eslint/unified-signatures -- for distinguishing deprecated constructor
+    error: any
+  );
```

Comment:
[NOTE]

Some ngrx actions were not refactored in the past PR https://github.com/SAP/spartacus/pull/17657/files from `any` to `ErrorActionType`. 

So in this PR I'm refactoring them directly from `any` to `ActionErrorProperty`.

Note: I expect in the upcoming weeks in `develop` there might be more **new** ngrx actions with  `error: any` that we should refactor analogically, before releasing our "Ssr Error Handling" epic